ddp-workspace/projects/ddp-circadia/src/app/components/user-activities/user-activities.component.scss

```diff
@@ -7,23 +7,30 @@
 .mat-cell,
 .mat-header-cell {
   font-size: 1.6rem;
-}
 
-.mat-cell {
-  padding-top: 1.6rem;
-  padding-bottom: 1.6rem;
+  padding-top: 1.6rem !important;
```

Comment:
#minor consider increasing the specificity via component class, but not via !important;